Apromore-Extras/APMLogModule/src/main/java/org/apromore/apmlog/stats/CaseAttributeValue.java

```diff
@@ -92,7 +105,7 @@ public double getOppCases() {
     }
 
     public double getPercent() {
-        return percent;
+        return 100 * ((double) getOccurCasesIndexSet().size() / totalCases);
     }
```

Comment:
The fact that .getFrequency and .getPercent are really the same value formatted different ways isn't obvious from the names.  Perhaps something like:
```
public double getFrequency() {
    return ((double) getOccurCasesIndexSet().size()) / totalCases;
}

static String formatAsPercentage(double n) {
    return String.format("%.2f", 100 * n);
}

public String getFrequencyFormattedAsPercentage() {
    return formatAsPercentage(getFrequency());
}
```

Reply:
they are the interfaces for ZUL.

Reply:
That's a good reason.